### ISSUE TITLE
Implement Levenshtein Distance

### ISSUE THREAD

**Issue by saurabh47** [saurabh47/Popular_Coding_Questions_Solution#8]
Write a function that takes in two strings and returns the minimum number of edit operations that need to be performed on the first string to obtain the second string.

There are three edit operations: insertion of a character, deletion of a character, and substitution of a character for another.

**Sample Input**
```
str1 = "abc"
str2 = "yabd"
```
**Sample Output**
```
2 // insert "y"; substitute "c" for "d"
```

**Comment by Nithin005** [comment]
can i do this?

**Comment by saurabh47** [comment]
@Nithin005 yes, solve the problem and send the pull request. we will review it.

**Comment by Nithin005** [comment]
which programming language should i use?

**Comment by saurabh47** [comment]
@Nithin005 you can use any programming language.

**Comment by maheshmnj** [comment]
@Nithin005 while submitting the PR include a README file which guides on how to run the program and also include if any specific  runtime environment is required to run the program
